pkg/util/constants.go

```diff
@@ -229,3 +229,8 @@ var (
 	// EndpointSliceGVK is the GroupVersionKind of K8s native EndpointSlice.
 	EndpointSliceGVK = discoveryv1.SchemeGroupVersion.WithKind("EndpointSlice")
 )
+
+const (
+	// PVCSelectedNodeAnnotation is the annotation key for the node mounting the persistent volume claim.
+	PVCSelectedNodeAnnotation = "volume.kubernetes.io/selected-node"
```

Comment:
Can we directly reference constants in the Kubernetes repo?

Reply:
sure, but we have to vendor more files

Reply:
Will there be a lot of vendor files added? If it's less, I feel it's acceptable.

Reply:
I went ahead and pushed up a second commit for you to assess. Let me know if you want me to squash or revert the commit